tests/test_index_old.cpp

```diff
@@ -13,18 +13,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "simd/simd.h"
```

Comment:
function test cases should not depend on any internal definitions or interfaces, so don't include header files from src/

Reply:
some basic func like l2sqr, innerproduct is common in functest & internal, If I write it twice, is it a bit redundant? The correctness of the simd module has been guaranteed by unittest.